templates/4a_compensate_image_data.ipynb

```diff
@@ -14,7 +14,7 @@
    "id": "237267d0",
```

Comment:
<p>This part is still a bit confusing. Can ask Ben/Hadeesha/Cameron to weigh in</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>I think it is hard to say, this seems clear to me, but maybe only because I have gone through the process, Maybe say something of the lines of "try new multiplier values until you find a multiplier that is able to give you your desired images." Just so someone knows what it is they are trying to do with this step.</p>